en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md

```diff
@@ -32,54 +42,224 @@ https://console.choreo.dev/login/</a>.
+
+##  Configuring ELK Analytics
+
+Follow the instructions below to configure ELK Analytics for Choreo Connect:
+
+[![Choreo ELK Analytics Data Flow]({{base_path}}/assets/img/deploy/choreo-elk-analytics-data-flow.png)]({{base_path}}/assets/img/deploy choreo-elk-analytics-data-flow.png)
+
+The Enforcer component in Choreo Connect can log analytics-related data to be used by Filebeat in ELK Stack. Data flow for the ELK Analytics can be depicted as given above.
+
+### Step 1 - Configure Choreo Connect
+
+#### Step 1.1 - Prepare the reporter JAR
+
+1. Checkout and build the [wso2/samples-apim](https://github.com/wso2/samples-apim) repository.
+
+2. Navigate to the `analytics-event-publisher/target` directory.
+3. Copy and paste the generated JAR in to the `choreo-connect-1.x.x/docker-compose/resources/enforcer/dropins` directory. 
+     
+     This JAR is added to the Java classpath when the Enforcer starts.
+
+#### Step 1.2 - Configure the config.toml file
+
+1. Open the `choreo-connect-1.x.x/docker-compose/choreo-connect(-with-apim)/conf/config.toml` file.
+2. Update the analytics section based on the following configurations.
 
     ``` toml
     [analytics]
-    enabled = true
-    [analytics.adapter]
-        bufferFlushInterval = "1s"
-        bufferSizeBytes = 16384
-        gRPCRequestTimeout = "20s"
+        enabled = true
+       
     [analytics.enforcer]
         [analytics.enforcer.configProperties]
-        authURL = "$env{analytics_authURL}"
-        authToken = "$env{analytics_authToken}"
-
-        [analytics.enforcer.LogReceiver]
-        port = 18090
-        maxMessageSize = 1000000000
-        maxHeaderLimit = 8192
-        #keep alive time of the external authz connection
-        keepAliveTime = 600
-
-        [analytics.enforcer.LogReceiver.threadPool]
-            coreSize = 10
-            maxSize = 100
-            #keep alive time of threads in seconds
-            keepAliveTime = 600
-            queueSize = 1000
+            "publisher.reporter.class" = "org.wso2.am.analytics.publisher.sample.reporter.CustomReporter"
     ```
+   
+!!! note "If you want to use a custom reporter class"
+    If you want to use a custom reporter class, follow the instructions below:
+     
+    1. Compile the new reporter implementation as a JAR file.</br>
+    2. Mount it to the `/home/wso2/lib/dropins` directory within the Enforcer by adding the JAR file to the choreo-connect-1.x.x/docker-compose/resources/enforcer/dropins directory.</br>
+    3. Set the value of the `publisher.reporter.class` property to the class name of the new reporter implementation in the `config.toml` file as follows:
+        
+        ``` toml
+        [analytics]
+            enabled = true
 
-### Step 2 - Try it out
+        [analytics.enforcer]
+            [analytics.enforcer.configProperties]
+                "publisher.reporter.class" = "org.wso2.am.analytics.publisher.sample.reporter.CustomReporter"
+        ```
```

Comment:
This note is almost same as step 1.1 and step 1.2. Sall we only mention as a reminder to update the publisher.reporter.class as the customer reporter they use.